src/AasxServerBlazor/AasxServerBlazor.csproj

```diff
@@ -1,82 +1,86 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Reformated this file, because some lines used spaces and others tabs as delimiter. Found that kind of ugly. I can revert that, if needed.